sandbox/api/constants.py

```diff
@@ -29,12 +28,51 @@
 RELATED_IDENTIFIERS = ["9000000009", "9000000025"]
 
 CONSENT_PERFORMER = "Consent:performer"
+CONSENT_PATIENT = "Consent:patient"
 
 # Example files
 
 # Common examples
 INTERNAL_SERVER_ERROR_EXAMPLE = "./api/examples/errors/internal-server-error.yaml"
+BAD_REQUEST_INCLUDE_PARAM_INVALID = (
+    "./api/examples/errors/invalid-include-parameter.yaml"
+)
+NOT_FOUND = "./api/examples/errors/not-found.yaml"
+MISSING_IDENTIFIER = "./api/examples/errors/missing-identifier.yaml"
+INVALID_IDENTIFIER = "./api/examples/errors/invalid-identifier.yaml"
+
 # Consent examples
-CONSENT__ADULT_CONSENTING_EXAMPLE = "./api/examples/GET_Consent/adults-consenting.yaml"
-CONSENT__MIXED_EXAMPLE = "./api/examples/GET_Consent/mixed.yaml"
-CONSENT__MOTHER_CHILD_EXAMPLE = "./api/examples/GET_Consent/mother-child.yaml"
+consent_dir = "./api/examples/GET_Consent/"
+CONSENT__FILTERED_RELATIONSHIPS_STATUS_ACTIVE = (
```

Comment:
In future try to use f strings as it handles a lot of type errors. Not a problem for this as both are string. F strings also improves readability